api/models.py

```diff
@@ -375,6 +375,9 @@ class AssessmentType(models.IntegerChoices):
     assessment_type = models.IntegerField(verbose_name=_("Country Assessment Type"), choices=AssessmentType.choices)
     branch_name = models.CharField(verbose_name=_("Branch Name"), max_length=255, null=True, blank=True)
 
+    class Meta:
+        unique_together = ("country", "assessment_code", "year")
```

Comment:
Missing `assessment_type`

Also, looking at the source data,
assessment_code = 
 - For ocacpublic: NS_ID+Year
 - For bocapublic: NS_ID+Branch_Name+Year
 Where assessment_code already includes country and year.
 
 Maybe should use ("assessment_code", "assessment_type") as unique?